projects/packages/videopress/src/client/components/video-upload-area/style.module.scss

```diff
@@ -0,0 +1,64 @@
+.wrapper {
+	max-width: 552px;
+	width: 100%;
+	border-radius: var(--jp-border-radius);
+	border: 1px dashed var(--jp-gray-20);
+	background-color: var(--jp-gray-0);
+	padding: 48px 0 64px;
+
+	.button {
+		background: var(--jp-white);
+		padding: 4px 8px;
+		font-size: var(--font-body-extra-small);
+	}
+
+	&.small {
+		max-width: 358px;
+		padding: 19px 0 44px;
+	}
+}
+
+.hover {
+	background-color: var(--jp-gray-5);
+	transition: all 0.1s linear;
+
+	.button:disabled {
+		background-color: #E7E7E8;
+		color: #9A9A9B;
+		box-shadow: inset 0 0 0 1px #9A9A9B;
+	}
```

Comment:
The same point about overriding styles, and in this case, you should use `disabled` prop: https://automattic.github.io/jetpack-storybook/?path=/story/js-packages-components-button--disabled